### PR TITLE
ci: Add CentOS dependencies for CRI-O shimv2

### LINKED ISSUE #2654: Add CentOS dependencies for CRI-O

### PR DESCRIPTION
**Pull request by GabyCT**
This PR adds the packages dependencies that are needed to build and
install CRI-O for shimv2.

Fixes #2654

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: .ci/setup_env_centos.sh [hunk 1]
```diff
@@ -105,6 +105,16 @@ main()
 	# GNU parallel not available in Centos repos, so build it instead.
 	build_install_parallel
 
+	# Extra CRI-O dependencies
+	if [ "$centos_version" >= "8" ]; then
+		curl -OkL http://mirror.centos.org/centos/7/os/x86_64/Packages/btrfs-progs-4.9.1-1.el7.x86_64.rpm
+		sudo -E rpm -ivh btrfs-progs-4.9.1-1.el7.x86_64.rpm
```
Comment (devimc) [inline .ci/setup_env_centos.sh:111]: oii, why don't use the btrfs for centos 8? https://centos.pkgs.org/8/elrepo-testing-x86_64/

Comment (GabyCT) [reply .ci/setup_env_centos.sh:111]: @devimc centos 8 at http://mirror.centos.org/centos/8/ does not have anymore the btrfs

Comment (GabyCT) [reply .ci/setup_env_centos.sh:111]: @devimc the `elrepo-testing` is a third party repo

Comment (jcvenegas) [reply .ci/setup_env_centos.sh:111]: @GabyCT what other filesystem could we use instead of btrfs ?

Comment (GabyCT) [reply .ci/setup_env_centos.sh:111]: @jcvenegas `btrfs` is a dependency to build CRI-O, I am not sure if there is another filesystem that we could use

Comment (fidencio) [reply .ci/setup_env_centos.sh:111]: @GabyCT, we can simply skip btrfs without issues, just pass `exclude_graphdriver_btrfs` to the BUILDTAGS.

**Comment by fidencio** [comment]
@GabyCT, lgtm. I'd just add one mention in the commit message that EL8 doesn't support btrfs.

**Comment by GabyCT** [comment]
@fidencio thanks, I modify the buildtags

**Pull request merged**